api/src/main/java/gov/cms/ab2d/api/service/JobServiceImpl.java

```diff
@@ -36,4 +37,13 @@ public Job createJob(String resourceTypes, String url) {
 
         return jobRepository.save(job);
     }
+
+    public void cancelJob(String jobId) {
+        Job job = jobRepository.findByJobID(jobId);
+        if (job == null) {
+            throw new EntityNotFoundException("No job with jobID " +  jobId + "  was found");
+        }
+
```

Comment:
We need to throw an exception if a user is trying to cancel a job that's in FAILED or COMPLETE status.